aws/resource_aws_subnet_test.go

```diff
@@ -616,12 +646,21 @@ resource "aws_subnet" "test" {
   assign_ipv6_address_on_creation = false
 
   tags = {
-    Name = "tf-acc-subnet-ipv6-update-cidr"
+  	Name = "tf-acc-subnet-ipv6-update-cidr"
   }
 }
 `
 
 const testAccSubnetConfigAvailabilityZoneId = `
+data "aws_availability_zones" "test" {
+  state = "available"
+
+  filter {
+    name   = "opt-in-status"
+    values = ["opt-in-not-required"]
+  }
+}
```

Comment:
Let's drop in the standard `composeConfig(testAccAvailableAZsNoOptInConfig(), ...)` here 👍

Reply:
done